backend/controllers/admin/admins.js

```diff
@@ -4,10 +4,22 @@ const ResponseHelper = require("../../helpers/response");
 
 module.exports = {
   getAdmins: async (req, res) => {
-    const admins = await User.findAll({ where: { user_type: "admin" } });
+    const page = +req.query.page || 1;
+    const limit = +req.query.limit || 5;
+    const offset = limit * (page - 1);
+
+    const count = await User.count({ where: { user_type: "admin" } });
+    const admins = await User.findAll({
+      limit,
+      offset,
+      where: { user_type: "admin" },
+    });
```

Comment:
- probably will work. but can you try a more simplier one, maybe possible in just one query only :thinking: 
- :eye: https://stackoverflow.com/questions/38211170/sequelize-pagination